src/components/IdleStatusMonitor.jsx

```diff
@@ -0,0 +1,183 @@
+import _ from 'lodash/fp';
+import React, { useEffect, useState } from 'react';
+import { lastActiveTimeStore } from 'libs/state'; // stub
+import * as Utils from 'libs/utils';
+import PropTypes from 'prop-types';
+import {
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Button,
+  withStyles,
+} from '@material-ui/core';
+
+// Styles
+const styles = (theme) => ({
+  buttonBar: {
+    marginTop: '1rem',
+    display: 'flex',
+    alignItem: 'baseline',
+    justifyContent: 'flex-end',
+  },
+  timer: {
+    color: theme?.palette.primary.dark,
+    whiteSpace: 'pre',
+    textAlign: 'center',
+    fontSize: '4rem',
+  },
+});
+
+// Helpers
+const displayRemainingTime = (remainingSeconds) =>
+  _.join(':', [
```

Comment:
We have the `moment` library imported.  Feels like we should be using that for format durations/times, etc. (https://momentjs.com/)

Reply:
Switched to `moment` - Thanks!